.gitignore

```diff
@@ -12,3 +12,4 @@ modules/[Aa][Hh][Kk]_[Tt]ests
 .settings
 .vscode
 sdk/tools/winesync/winesync.cfg
+.dccache
```

Comment:
Delete this line and delete the file .dccache

Reply:
ok
but this file is generated every time i change files content